Ch02/Ch02.md

```diff
@@ -101,117 +101,116 @@ that we have available. This is useful for quick checks and verifications, but i
 more serious than that. 
 
 Obviously, all data in the live instance is public, and there are no guarantees about availability. We use this 
-instance to tryout the latest versions, so that is also a consideration. In short, if you need to verify something 
+instance to try out the latest versions, so you should take that into consideration. In short, if you need to verify something 
 small, go ahead and hit that instance. Otherwise, you'll need your own version.
 
 ## Your first database
 
-By now, you have already setup an instance of RavenDB to work with, and managed to load the RavenDB Studio in 
-your browser. For simplicity's sake, I'm going to assume from now on that you are running on the local machine
-on port 8080. Point your browser to [http://localhost:8080](http://localhost:8080) and you should be greeted with
+At this point, you've already set up an instance of RavenDB to work with, and you've loaded the RavenDB Studio in 
+your browser. For simplicity's sake, I'm going to assume from now on that you're running RavenDB on the local machine
+on port 8080. Point your browser to [http://localhost:8080](http://localhost:8080), and you should be greeted with
 an empty RavenDB instance. You can see how it looks in Figure 2.1.
 
 ![Empty RavenDB node](./Ch02/img01.png)
 
-What we have right now is a RavenDB node that is a self-contained cluster^[Actually, that is not really the case, 
+What we have right now is a RavenDB node that is a self-contained cluster^[Actually, that's not exactly the case, 
 but the details on the state of a newly minted node are a bit complex and covered in more detailed in 
-[Your first RavenDB Cluster](#clustering-intro) chapter].
+["Your First RavenDB Cluster"](#clustering-intro) chapter].
 Now that we have a running node, the next stage it to create a new database on this node. 
 
-You can do that by clicking on the `New database` button, name it `Northwind`, and accept all the defaults for
-the dialog. We'll discuss what all of those mean in [Chapter 19](#prod-deploy). Click on the `Create` button and 
-that is pretty much it, your new database is ready, you can see how this looks like in Figure 2.2.
+You can do that by clicking the `New database` button, naming it `Northwind` and accepting all the defaults in
+the dialog. We'll discuss what all of those mean in [Chapter 19](#prod-deploy). Click the `Create` button, and 
```

Comment:
Do we also need quotes here? Probably better to use the chapter name to be consistent.